api/src/main/java/com/velocitypowered/api/network/ProtocolVersion.java

```diff
@@ -13,6 +13,7 @@
 public enum ProtocolVersion {
   UNKNOWN(-1, "Unknown"),
   LEGACY(-2, "Legacy"),
+  MINECRAFT_1_7(5, "1.7"),
```

Comment:
1.7 - 1.7.10 have multiple protocol versions (1.7-1.7.5, 1.7.6-1.7.10 if i not mistaken)